sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -145,20 +145,19 @@ public virtual Response<TimeSeriesModelSettings> GetModelSettings(CancellationTo
         }
 
         /// <summary>
-        /// Updates model name or default type Id on Time Series model settings asynchronously.
+        /// Updates model name on Time Series model settings asynchronously.
         /// </summary>
-        /// <param name="options">Model settings update request body.</param>
+        /// <param name="name">Model display name which is shown in the UX and mutable by the user. Initial value is &quot;DefaultModel&quot;.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The updated model settings with the http response <see cref="Response{TimeSeriesModelSettings}"/>.</returns>
-        /// <exception cref="ArgumentNullException"> <paramref name="options"/> is null. </exception>
-        public virtual async Task<Response<TimeSeriesModelSettings>> UpdateModelSettingsAsync(UpdateModelSettingsOptions options, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<TimeSeriesModelSettings>> UpdateModelSettingsNameAsync(string name, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(UpdateModelSettings)}");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TimeSeriesInsightsClient)}.{nameof(UpdateModelSettingsName)}");
             scope.Start();
             try
             {
                 // To do: Generate client session Id
-                Argument.AssertNotNull(options, nameof(options));
```

Comment:
Maybe keep the assertNotNullOrEmpty for the name parameter

Reply:
In update call in ModelSettingsRestClient I see this. 

/// <exception cref="ArgumentNullException"> <paramref name="parameters"/> is null. </exception>